__tests__/index.test.js

```diff
@@ -31,3 +31,36 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("status 200: responds with an object containing all endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        for (let key in body.endpoints) {
+          expect(typeof body.endpoints[key].description).toBe("string");
```

Comment:
This is very thorough and valid, but an easier way might have been to use a .toEqual against the file we are using with the endpoints

Reply:
Thank you, now I got it, it seems I didn't understand quite well the task!